myapp/spec/system/tasks_spec.rb

```diff
@@ -164,14 +193,19 @@
   end
 
   describe '#show' do
+    before do
```

Comment:
I want to add a test to ensure that users cannot see other users' tasks.

Reply:
I added some test scenarios for #show and #edit as well.